src/Deployer.php

```diff
@@ -58,15 +59,28 @@ class Deployer
     private $stageStrategy;
 
     /**
+     * @var Emitter
+     */
+    private $emitter;
+
+    /**
+     * Deployer constructor.
+     *
      * @param Application $console
      * @param Console\Input\InputInterface $input
      * @param Console\Output\OutputInterface $output
-     */
-    public function __construct(Application $console, Console\Input\InputInterface $input, Console\Output\OutputInterface $output)
-    {
+     * @param Emitter $emitter
+     */
+    public function __construct(
+        Application $console,
+        Console\Input\InputInterface $input,
+        Console\Output\OutputInterface $output,
+        Emitter $emitter
```

Comment:
may better set emiiter via setter? because it is not a mandatory property. Deployer will still work as expected without emitter. Usually only required properties are passed via constructor.